tools/smartdnsblock/smartdnsblock/smartdnsblock.cpp

```diff
@@ -35,15 +35,31 @@ ULONG GET_ADAPTERS_ADDRESSES_BUFFER_SIZE = 16384;
 PCWSTR FILTER_PROVIDER_NAME = L"Outline";
 PCWSTR SUBLAYER_NAME = L"Smart DNS Block";
 
+UINT64 MAX_TRIES = 3;
```

Comment:
Should this be 2? From the documentation, it seems like it would be enough to recover form the buffer overflow.